src/main/java/com/notably/model/block/Body.java

```diff
@@ -29,7 +35,7 @@ public Body(String body) {
      * @param test The body string to test.
      */
     public static boolean isValidBody(String test) {
-        return !test.isBlank();
+        return !test.matches(VALIDATION_REGEX);
```

Comment:
Wouldn't it seem rather counter-intuitive that the `VALIDATION_REGEX` here actually matches the invalid scenario? It sort of breaks the convention of the other classes, where `VALIDATION_REGEX` always contains the regex that verifies for correctness.

Reply:
Understood! Will modify the `VALIDATION_REGEX`